### ISSUE TITLE
Doomguard - Cripple

### ISSUE THREAD

**Issue by Xadras** [Looking4Group/L4G_Core#2775]
Originally reported by: **Selphius (Bitbucket: [Selphius](https://bitbucket.org/Selphius), GitHub: [Selphius](https://github.com/Selphius))**

----------------------------------------

I can't believe it, that people to this day didn't report such critical bug and even dare to use it (80% alliance guilds, no idea about horde). And later you cry "PvE" is too easy...

So dear devs, no idea if you knew about it or not, but warlock's Doomguard has skill called "http://wow.gamepedia.com/Cripple" it shouldn't work on bosses, it's hard to find any conclusive evidence of it since no one write about it at all. I can only provide from my personal experience and some stuff related to it.

http://www.excalibur-nw.com/forum/tracker/issue-2739-raid-bosses-are-not-immune-to-cripple/
In this link you can see excalibur devs didn't gave a damn about this issue at all...

Another link: it's from WoTLK but comment here relates to ALL expansions.
http://www.arenajunkies.com/topic/72324-doomguard-hotfixed/

Quote: "Cripple worked on raid bosses"
Answer: No, almost all were immune.  I vaguely remember it working on Grobbulus, but that's about it.  It did work on most trash, however. 

Also some wowhead comments related to this summon, if it worked they wouldn't say things like this.
http://www.wowhead.com/npc=11859/doomguard#comments

"By Bengarion (8,860 – 2·19·57) on 2007/12/14 (Patch 2.3.0)" 
Is there a practical use for a doomguard - or is it just a toy for warlocks to unleash in capitol cities for fun and destruction?
"truckman1 on 2008/11/06 (Patch 3.0.3)
the latter

To sum it up, there is a lot of difference in opinions: one's say it works on bosses and it should, some not. But if it really should work, then a lot of bosses dependent on attack speed WILL BE A JOKE, Mother Shahraz, Brutallus etc. 
If you don't want PvE to be a joke cause of it, you must take this one seriously.

----------------------------------------
- Bitbucket: https://bitbucket.org/looking4group_b2tbc/looking4group/issue/2681

**Comment by AnonXS** [comment]
https://github.com/Looking4Group/L4G_Core/commit/ca8a1b3109f8e9b32442318796f964b25903b749
